versions.tf

```
@@ -4,5 +4,6 @@ terraform {
   required_providers {
     local  = "~> 1.2"
     random = "~> 2.2"
+    aws    = "~> 3.0"
```

@aknysh Should we stick to AWS 2.0 for now? Maybe 
```suggestion
    aws    = ">= 2.0, < 4.0"
```

---

yes we should support 0.12 as well, and all aws providers.

---

@Nuru @aknysh updated